publication-commons/src/main/java/no/unit/nva/publication/model/business/logentry/FileLogEntry.java

```diff
@@ -0,0 +1,76 @@
+package no.unit.nva.publication.model.business.logentry;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import java.time.Instant;
+import no.unit.nva.identifiers.SortableIdentifier;
+import no.unit.nva.publication.service.impl.ResourceService;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public record FileLogEntry(SortableIdentifier identifier, SortableIdentifier fileIdentifier,
```

Comment:
This signature is very easy to mess up when using; I'd be tempted to force use of the builder